python3/koans/about_comprehension.py

```diff
@@ -55,7 +55,7 @@ def test_creating_a_dictionary_with_dictionary_comprehension(self):
 
         dict_comprehension = { k.upper(): weapon for k, weapon in dict_of_weapons.items() if weapon}
 
-        self.assertEqual(__, 'first' in dict_comprehension)
-        self.assertEqual(__, 'FIRST' in dict_comprehension)
-        self.assertEqual(__, len(dict_of_weapons))
-        self.assertEqual(__, len(dict_comprehension))
+        self.assertEqual(False, 'first' in dict_comprehension)
+        self.assertEqual(False, 'FIRST' in dict_comprehension)
```

Comment:
how is this test passing?

this assert is ok
```
self.assertEqual(False, 'first' in dict_comprehension)
```

this assert should be failing
```
self.assertEqual(False, 'FIRST' in dict_comprehension)
```

Reply:
Yes, this test is failing. It should be True instead of False.

self.assertEqual(False, 'FIRST' in dict_comprehension)